src/lib/comms/include/sol-http.h

```diff
@@ -236,9 +250,108 @@ sol_http_param_init(struct sol_http_param *params)
 
 bool sol_http_param_add(struct sol_http_param *params,
     struct sol_http_param_value value) SOL_ATTR_WARN_UNUSED_RESULT;
+bool sol_http_param_add_copy(struct sol_http_param *params, struct sol_http_param_value value);
 void sol_http_param_free(struct sol_http_param *params);
 
-int sol_http_escape_string(char **escaped, const char *value);
+/**
+ * Encodes an URL string.
+ *
+ * If the string was not encoded this function will simple copy
+ * the slices content to the buffer and set @c SOL_BUFFER_FLAGS_MEMORY_NOT_OWNE.D
```

Comment:
there is a "." in the flag name.

Also check Doxygen generated file, I think that you can use # before the name to link to that flag, like `#SOL_BUFFER_FLAGS_MEMORY_NOT_OWNED`